### DISCUSSION TITLE
How do I screen?

### DISCUSSION THREAD

**Discussion by Fennavanommen** [asreview/asreview#930] [Q&A]
For a systematic review that I am conducting with my research team I have a total of around 9000 articles. I screened 1000 articles in ASReview lab and received a ranking after this. Do I still need to screen all 9000 articles or is there a threshold for when I can stop screening?  

Does anyone have any experience with this? Let me know.

**Comment by FelixWdm** [comment]
Hi @Fennavanommen, at this point we can't give a single treshold to be used. 

It could be you missed some papers, could be you found them all by screening the first 1000. We can give you some guidance though. You use ASReview to improve the sorting of the remaining articles, each screening decision helps the algorithm in determining the next most likely to be relevant paper for you. So you do expect to see an increase in the percentage of relevant papers (compared to random order) during the screening. If at some point you see more irrelevant papers coming up, it could be you've found all that could be marked relevant. But there's no guarantee, there could be an odd article you haven't seen yet.
Did you during the search process also use a more precise search, so more focused, with less results? If so the number found with that search might also be an indication of the total number of relevant papers 'out there'. 

And also keep in mind, searching databases and screening the result is not the only method to find relevant articles. Always incorparate citation search/snowball searching as well, to find articles missed by your search or screening process.

I hope this helps, if not, please reply.

Best Felix

**Comment by Rensvandeschoot** [comment]
The idea of ASReview is to save time, so the entire set-up is to stop with screening BEFORE you have reached the end of your database. When to stop reviewing can be based on: (1) your time is up or (2) you only see irrelevant suggestions for already many iterations in a row.
